src/main/kotlin/no/nb/mlt/wls/domain/WLSService.kt

```diff
@@ -183,7 +154,7 @@ class WLSService(
         hostOrderId: String
     ) {
         val order = getOrderOrThrow(hostName, hostOrderId)
-        order.throwIfInProgress()
+        order.deleteOrder()
```

Comment:
I think I already mentioned it, but a change of name would be good here, or change how this is handled